app/screens/edit_post/edit_post.js

```diff
@@ -228,7 +227,6 @@ export default class EditPost extends PureComponent {
                             multiline={true}
                             numberOfLines={10}
                             style={[style.input, {height}]}
-                            autoFocus={true}
```

Comment:
by removing this, when you enter the screen,.. does the text input receives focus or do you need to tap on it?